test/unit/ComponentConfig.test.js

```diff
@@ -145,14 +145,20 @@ describe('The ComponentConfig class', function() {
         });
     });
 
-    describe('ComponentConfig.normalizeConfiguration(app, config):', function(){
-        it('instantiates a ComponentConfig and calls the normalize method', function(){
+    describe('ComponentConfig.normalizeConfiguration(app, config):', () => {
+
+        it('instantiates a ComponentConfig and calls the normalize method', function() {
             const dataSource = 'testDataSource';
             const exposeAt = 'some-property';
             const maxDelay = 3000;
             const models = {
-                Test: true,
-                Dummy: false,
+                ConfigFalse: false,
+                ConfigTrue: true,
+                ConfigPublic: {
+                    expose: true,
+                    isPublic: true,
+                },
+                ConfigEmpty: {},
```

Comment:
You could add some negative/mixed up cases for the options one may provide.

Reply:
It actually covers most of the cases. Nevertheless I can add some more cases (but note, I'am still working on a task which is has no planning or budget).